flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/utils/InternalConfigOptions.java

```diff
@@ -50,4 +51,13 @@
                                     + " as UTC+0 milliseconds since epoch for simplification, this config will be used by"
                                     + " some temporal functions like LOCAL_TIMESTAMP in batch job to make sure these"
                                     + " temporal functions has query-start semantics.");
+
+    @Experimental
+    public static final ConfigOption<Boolean> TABLE_EXEC_NON_TEMPORAL_SORT_ENABLED =
+            key("table.exec.sort.non-temporal.enabled")
```

Comment:
Any reason for not using the notation `__` like in the other 2 config options above?

Reply:
No, I can do, I'm just waiting for some confirmation from @wenlong88 here, if maybe we can completely remove this option, as it's only used for test.

Reply:
Ok, so it's ok to move it to the InternalOptions and the test usage is still valid.